gemrb/includes/win32def.h

```diff
@@ -18,51 +18,22 @@
  *
  */
 
-/**
- * @file win32def.h
- * Some global definitions, mostly for Un*x vs. MS Windows compatibility
- * @author The GemRB Project
- */
-
+// TODO: move this to platforms/windows
 
 #ifndef WIN32DEF_H
 #define WIN32DEF_H
 
-#include "exports.h"
-
-#include "System/String.h"
-
-#ifdef WIN32
-# define WIN32_LEAN_AND_MEAN
-
-# ifndef NOMINMAX
-# define NOMINMAX
-# endif
-
+#define WIN32_LEAN_AND_MEAN
+#define NOMINMAX
```

Comment:
Well, we need this conditionally, otherwise MSYS2 complains:
```
C:/MSYS2/mingw64/include/c++/10.2.0/x86_64-w64-mingw32/bits/os_defines.h:45: note: this is the location of the previous definition
   45 | #define NOMINMAX 1
```